lib/src/wiggle_abi/req_impl.rs

```diff
@@ -89,6 +89,16 @@ impl FastlyHttpReq for Session {
         }
     }
 
+    #[allow(unused_variables)] // FIXME FGS 2023-06-14: Remove this directive once implemented.
+    fn downstream_client_request_id(
+        &mut self,
+        reqid_out: &GuestPtr<u8>,
+        reqid_max_len: u32,
+        nwritten_out: &GuestPtr<u32>,
+    ) -> Result<(), Error> {
+        Ok(())
```

Comment:
We should probably at least write 0 to `nwritten_out`.

Reply:
Done.